src/librustc_error_codes/error_codes/E0754.md

```diff
@@ -1,31 +1,27 @@
 An non-ascii identifier was used in an invalid context.
 
-Erroneous code example:
+Erroneous code examples:
 
 ```compile_fail,E0754
 # #![feature(non_ascii_idents)]
 
 mod řųśť;
 // ^ error!
-fn main() {}
-```
-
-```compile_fail,E0754
-# #![feature(non_ascii_idents)]
 
 #[no_mangle]
 fn řųśť() {}
 // ^ error!
+
 fn main() {}
 ```
 
-Non-ascii can be used as module names if it is inline
-or a #\[path\] attribute is specified. For example:
+Non-ascii can be used as module names if it is inline or a `#[path]` attribute
```

Comment:
`#[path]` attribute? What is that, should we do a link to that page?

Reply:
Absolutely! As for the `path` attribute, it allows you to link to a file module by its system path and give it a different name.

Reply:
Yes, but should we link it to the reference to let the users know what is that? I feel like this is a less known item.

Reply:
The problem with external links is that they might change (for example in this case, this is: https://doc.rust-lang.org/reference/items/modules.html#the-path-attribute), even if they're part of the rust book.

Reply:
But don't we have link check?

Reply:
Only when referring to items inside the docs scope (so relative paths, not absolute ones).